lib/global_uid.rb

```diff
@@ -16,19 +16,20 @@ class TimeoutException < Exception ; end
 ActiveRecord::Base.send(:include, GlobalUid::ActiveRecordExtension)
 ActiveRecord::Migration.send(:prepend, GlobalUid::MigrationExtension)
 
+# Make sure that GlobalUID is disabled for ActiveRecord's SchemaMigration table
 if defined?(ActiveRecord::SchemaMigration)
   ActiveRecord::SchemaMigration.disable_global_uid
 end
 
+# Make sure that GlobalUID is disabled for ActiveRecord's Internal Metadata table
+if ActiveRecord::VERSION::MAJOR >= 5
```

Comment:
I don’t know why my diff tool said line 6. I meant line 20, obviously. Why do we do two equivalent checks in two different ways, right after each other?